indexers/vector/FaissIndexer/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from typing import Tuple, Optional
 
+from faiss import normalize_L2
```

Comment:
Sorry, can you remove the import from here?

Reply:
Sorry, I had to rebase multiple times to rename the git commit messages. Now everything was merged wrongly. I'll check all the source code and push it again.